File: spec/python/specwrite/test_opaque_with_param.py
import unittest
from specwrite.common_spec import CommonSpec

from testwrite.opaque_with_param import OpaqueWithParam

class TestOpaqueWithParam(CommonSpec.Base):
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.struct_class = OpaqueWithParam
        self.src_filename = 'src/term_strz.bin'

File: spec/java/src/io/kaitai/struct/specwrite/TestOpaqueWithParam.java
package io.kaitai.struct.specwrite;

import io.kaitai.struct.KaitaiStruct.ReadWrite;
import io.kaitai.struct.testwrite.OpaqueWithParam;
import org.testng.annotations.Test;

public class TestOpaqueWithParam extends CommonSpec {
    @Override
    protected Class<? extends ReadWrite> getStructClass() {
        return OpaqueWithParam.class;
    }

    @Override
    protected String getSrcFilename() {
        return "term_strz.bin";
    }
}
